### PR TITLE
WIP - Add scripts support

### PR DESCRIPTION
**Pull request by devinrhode2**
This is VERY much a work-in-progress.

Opening early for visibility/feedback/etc

### CODE REVIEW

**Comment by justinfagnani** [comment]
Generally, I think issues are a great way to start the conversation. I'm happy to start on the PR.

The big thing to consider with scripts is security. I think it's probably a bad default to run scripts, certainly from a different origin. Is there an attribute to run scripts? Then, how does this work with CSP?

File: .eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/** @type {import('eslint').Linter.BaseConfig} */
```
Comment (justinfagnani) [inline .eslintrc.cjs:1]: let's keep the PR focused on one thing, and make project setup changes a different PR.

File: src/InjectRemoteHtmlInline/InjectRemoteHtmlInline.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from 'react'
```
Comment (justinfagnani) [inline src/InjectRemoteHtmlInline/InjectRemoteHtmlInline.tsx:1]: Why is there React code in this PR?

I didn't even include Lit in the first version because this component didn't need dependencies. I'd be *very* against including React. What's it for?

Comment (bennypowers) [reply src/InjectRemoteHtmlInline/InjectRemoteHtmlInline.tsx:1]: Is this PR LLM spam?

Comment (rictic) [reply src/InjectRemoteHtmlInline/InjectRemoteHtmlInline.tsx:1]: Too harsh Benny, IMO.

A React shim for <html-include> seems like it should live in a separate package, and possibly not in the html-include-element git repo. As this is written, it seems that  html-include would need to take a dependency on React, when currently it has zero deps.

**Comment by devinrhode2** [comment]
I won't delete this branch, in case anyone that stumbles upon this is also interested in adding some sort of scripts support.

File: src/InjectRemoteHtmlInline/InjectRemoteHtmlInline.tsx (hunk 2 above)
Comment (devinrhode2) [reply src/InjectRemoteHtmlInline/InjectRemoteHtmlInline.tsx:1]: Originally this was written in a react app, before I knew about html-include-element.

I'll use co-pilot for trivial suggestions, but that's it. I got burned a couple times on medium-size snippet suggestions and it was just the WORST!

Comment (devinrhode2) [reply src/InjectRemoteHtmlInline/InjectRemoteHtmlInline.tsx:1]: Never intended to suggest this project add react, but I know there's a lot of goofs out there that may frolic along thinking adding react to everything is fine and normal lol.

I think a react-shim doesn't even need to use html-include per se.

I'm actually going in a different direction, creating my own soup-to-nuts react shim for my projects specific use case. In that shim, I may create some options around `<ReactShadow>`, or let the consumer of some widget decide if they want to wrap it in `<ReactShadow>` or not.

Comment (devinrhode2) [reply src/InjectRemoteHtmlInline/InjectRemoteHtmlInline.tsx:1]: The shim will be for a solid-js based "micro-frontend"
